common/views/components/FilterDrawerRefine/FilterDrawerRefine.js

```diff
@@ -11,6 +11,124 @@ import Divider from '@weco/common/views/components/Divider/Divider';
 import { type SearchParams } from '@weco/common/services/catalogue/search-params';
 import { type CatalogueAggregationBucket } from '@weco/common/model/catalogue';
 
+const allWorkTypes = [
```

Comment:
Could we store this in the `data` folder?
That way we have a place we know we should probably be turning things into APIs?

Reply:
And is this called `workTypeAggregations`?